__tests__/posts.ts

```diff
@@ -499,8 +499,8 @@ describe('query post', () => {
     }
   }`;
 
-  it('should throw not found when cannot find post', () =>
-    testQueryErrorCode(client, { query: QUERY('notfound') }, 'NOT_FOUND'));
+  it('should throw not found when post was soft deleted', () =>
```

Comment:
Please add it as another test, don't replace the existing test 🙏

Reply:
No way. I was so excited to writing test first doing "TDD" and I immediately got it wrong 🤣